koan/Koan/Parser.hs

```diff
@@ -0,0 +1,73 @@
+module Koan.Parser where
```

Comment:
I mean in the code - i.e. "Now that you've set up the machinery, try using it to parse `blah`".